code/modules/mob/living/carbon/metroid/metroid.dm

```diff
@@ -925,6 +925,50 @@ mob/living/carbon/slime/var/temperature_resistance = T0C+75
 	if(!uses)
 		qdel(src)
 
+/obj/item/stack/tile/bluespace
+	name = "bluespace floor tile"
+	singular_name = "floor tile"
+	desc = "Through a series of micro-teleports these tiles let people move at incredible speeds"
+	icon_state = "tile-bluespace"
+	w_class = 3
+	force = 6
+	materials = list(MAT_METAL=500)
+	throwforce = 10
+	throw_speed = 3
+	throw_range = 7
+	flags = CONDUCT
+	max_amount = 60
+	turf_type = /turf/simulated/floor/bluespace
+
+
+/turf/simulated/floor/bluespace
+	slowdown = -1
+	icon_state = "bluespace"
+	desc = "Through a series of micro-teleports these tiles let people move at incredible speeds"
+	floor_tile = /obj/item/stack/tile/bluespace
+
+
+/obj/item/stack/tile/sepia
+	name = "sepia floor tile"
+	singular_name = "floor tile"
+	desc = "Time seems to flow very slowly around these tiles"

```

Comment:
Because the people who *actually* are virtually non-existent compared to those who are anal-retentively obsessed with it ;)